skype/skype-ps/skype/New-CsHuntGroup.md

```diff
@@ -218,24 +217,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -MusicOnHoldFileContent
-PARAMVALUE: Byte\[\]
+### -OverflowThreshold
+The OverflowThreshold parameter defines the number of simultaneous calls that can be in the queue at any one time before the overflow action is triggered. The OverflowThreshold can be any integer value between 0 and 200, inclusive. The default value is 50. A value of 0 causes the overflow action to be taken immediately.
```

Comment:
We can leave out the default value from the description.

Reply:
Fixed